lib/Gedmo/Translatable/Entity/Repository/TranslationRepository.php

```diff
@@ -117,13 +117,13 @@ public function findTranslations($entity)
         $wrapped = new EntityWrapper($entity, $this->_em);
         if ($wrapped->hasValidIdentifier()) {
             $entityId = $wrapped->getIdentifier();
-            $entityClass = $wrapped->getMetadata()->rootEntityName;
-            $translationMeta = $this->getClassMetadata(); // table inheritance support
-
             $config = $this
                 ->getTranslatableListener()
                 ->getConfiguration($this->_em, get_class($entity));
 
+            $entityClass = isset($config['useObjectClass']) ? $config['useObjectClass'] : $wrapped->getMetadata()->rootEntityName;
```

Comment:
I did originally, but then Issue84Test.php was failing saying 'useObjectClass' is not set.